rpcs3/rpcs3qt/update_manager.cpp

```diff
@@ -209,6 +209,12 @@ bool update_manager::handle_json(bool automatic, bool check_only, const QByteArr
 	m_expected_hash = latest[os]["checksum"].toString().toStdString();
 	m_expected_size = latest[os]["size"].toInt();
 
+	if (!m_request_url.starts_with("https://github.com/RPCS3/rpcs3"))
```

Comment:
Dangerous change, will break every old build if we're ever forced to do another build host migration such as the previous GitHub -> AppVeyor